app/forms/draft_collection_form.rb

```diff
@@ -39,9 +44,7 @@ class DraftCollectionForm < Reform::Form
   validate :reviewable_form
   validates :release_date, embargo_date: true
   validates :release_option, presence: true, inclusion: { in: %w[immediate delay depositor-selects] }
-  validates :release_duration, inclusion: {
-    in: ['1 month', '2 months', '6 months', '1 year', '2 years', '3 years']
```

Comment:
@peetucket this line is why I was inquiring re: `6 months` vs. `6 month`, but it might not make a different either way.

Reply:
Oh, hmmm... that is a mismatch to what was being set before in the options in the form itself.  I'll investigate.

Reply:
Here: https://github.com/sul-dlss/happy-heron/pull/867/files#diff-ec9c1247d60f9db33645bcaac1fbfb6f45fadb5841a990e6a9da52771222b0b5L82

Reply:
@peetucket ok, fine by me! thanks

Reply:
Ok, I think the actual values are not currently used, but will need to be soon, when we translate to actual embargo date at the work level, captured in a comment I made to a different ticket here: https://github.com/sul-dlss/happy-heron/issues/836#issuecomment-763860711

However, in order to be consistent with hydra_etd, I'm going with "6 months" (see https://github.com/sul-dlss/hydra_etd/blob/master/app/models/submission.rb#L110 and https://github.com/sul-dlss/hydra_etd/blob/main/app/views/submit/lightbox/_lb_embargo.html.erb#L1)